### ISSUE TITLE
Suggested changes to the ML training material

### ISSUE THREAD

**Issue by surbhigoel77** [Cambridge-ICCS/practical-ml-with-pytorch#51]
Proposed sequence of topics
1. How ML works
2. Steps involved in developing a model - loading data, preprocessing, model definition (algorithm, optimiser, loss), training, prediction)
3. Pytorch intro (data type it works with - tensors, other key concepts)
4. Types of problems - Regression and Classification
5. Hands-on with a Regression example
6. Hands-on with Classification example

Algorithm used: Neural Network

**Comment by surbhigoel77** [comment]
Pier Luigi's session requirements:

### Module imports:

import torch
import torch.nn as nn
import torch.optim as optim
from torch.utils.data import DataLoader, TensorDataset
from sklearn.model_selection import train_test_split
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import cartopy.crs as ccrs

### Tasks:
1. Load data from csv files
2. Dataframe of the data
3. Plot distribution of the variables
4. Training and Loss function
5. Visualisation using matplotlib and cartopy (maps of standard deviation, mean of differences between reference (ocean model) and ML output, correlation coefficient between reference and ML output.)

**Comment by jdenholm** [comment]
I'd be disinclined to call a neural network an algorithm, because it's just a function (once its domain has been defined). You might be better calling it a model. If you were fitting a polynomial function, for example, you wouldn't call it an algorithm, but it would be okay to call it a model.
